src/main/java/com/yura/springboot/lesson/service/GreetingService.java

```diff
@@ -1,14 +1,29 @@
 package com.yura.springboot.lesson.service;
 
+import com.yura.springboot.lesson.entity.CompanyEntity;
+import com.yura.springboot.lesson.repository.CompanyRepository;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
+
+import java.util.List;
+
 @Service
 public class GreetingService {
 
-    @Value("${app.services.greeting.message}")
-    private String greeting;
+    private final String greeting;
+    private final CompanyRepository companyRepository;
+
+    public GreetingService(CompanyRepository companyRepository,
+                           @Value("${app.services.greeting.message}") String greeting) {
```

Comment:
Safer to use constructor injection:
https://reflectoring.io/constructor-injection/